pkg/model/schema.go

```diff
@@ -109,15 +109,16 @@ func getStructKeys(t interface{}) map[string][]string {
 			for k, v := range getStructKeys(reflect.New(fieldType.Elem()).Interface()) {
 				result[k] = mergeAndSortUnique(result[k], v)
 			}
-		} else {
-			yamlTag := field.Tag.Get("yaml")
-			if yamlTag != "" && yamlTag != "-" {
-				parts := strings.Split(yamlTag, ",")
-				if len(parts) > 0 {
-					result[structFullName] = mergeAndSortUnique(result[structFullName], []string{parts[0]})
-				}
+		}
+
```

Comment:
Why removing the else? Does it have to be applied always now?

Reply:
this loop iterates over each property, then based on the type, it may generate a recursion, but it was only adding child properties, basically ignoring itself, resulting in missing parent properties